tests/unit/test_context7_client_extended.py

```
@@ -137,7 +139,7 @@ async def test_get_library_docs_success(self):
         mock_result = {"content": "# Library Docs\n\nTest documentation"}
         
         with patch.object(client, '_execute_tool', new_callable=AsyncMock, return_value=mock_result):
-            docs = await client.get_library_docs("/test/library", "test topic", 1000)
+            docs = await client.get_library_docs("/test/library", 1000, "test topic")  # pyright: ignore[reportGeneralTypeIssues]
```

Pylint (3.3.7)

[convention] Line too long (129/100)

(C0301)